.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,98 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["kind/bug"]
+assignees:
+  - github_user_1
+  - github_user_2
+  - github_user_3
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please, fill this form to help us improve the project.
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    attributes:
+      label: Current Behavior
+      description: Please describe below a concise explanation of what you're experiencing
+      placeholder: A concise description of what you're experiencing...
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: Please describe below a concise explanation of what you expected to happen.
+      placeholder: A concise description of what you expected to happen...
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Steps To Reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+        1. In this environment...
+        2. With this config...
+        3. Run '...'
+        4. See error...
+  - type: textarea
+    id: workaround
+    attributes:
+      label: Workaround (if any)
+      description: Please provide any manual steps that allow you to resolve the issue
+      placeholder: What are the steps you followed to resolve the issue?
+    validations:
+      required: false
+  - type: dropdown
+    id: openshift-version
+    attributes:
+      label: OpenShift Version
+      description: What version of OpenShift are you running?
+      options:
+        - Baremetal
+        - OpenStack
+        - RHV
+        - AWS
+        - OKD
+        - CodeReady Containers
+        - Other
+    validations:
+      required: false
+  - type: input
+    id: other-openshift-version
+    attributes:
+      label: Openshift Version
+      description: If selected other, which version of OpenShift are you running?
+      placeholder: e.g. GCP...
+    validations:
+      required: false
```

Comment:
We should separate this into two separate questions:
 - *Required* `What version of OpenShift?` <Version Number>
 - *Optional* `On what infrastructure is OpenShift running on, if known?` Baremetal, OSP, RHV, AWS, ...